site-src/guides/index.md

```diff
@@ -91,43 +91,47 @@ Tooling:
 
       ```bash
       export GATEWAY_PROVIDER=gke
+      export GAIE_CHART_VERSION=v1.0.1-rc.1
       helm install vllm-llama3-8b-instruct \
       --set inferencePool.modelServers.matchLabels.app=vllm-llama3-8b-instruct \
       --set provider.name=$GATEWAY_PROVIDER \
-      --version v1.0.1-rc.1 \
+      --version GAIE_CHART_VERSION \
       oci://registry.k8s.io/gateway-api-inference-extension/charts/inferencepool
       ```
 
 === "Istio"
 
       ```bash
       export GATEWAY_PROVIDER=istio
+      export GAIE_CHART_VERSION=v1.0.1-rc.1
```

Comment:
can we rename the var to IGW_CHART_VERSION in all places? (shortcut for Inference Gateway)
```suggestion
      export IGW_CHART_VERSION=v1.0.1-rc.1
```